code/datums/webhook.dm

```diff
@@ -0,0 +1,71 @@
+var/list/global_webhooks = list()
+
+/hook/startup/proc/initialize_webhooks()
+	return global_initialize_webhooks()
+
+proc/global_initialize_webhooks()
+	var/list/Lines = file2list("config/webhooks.txt")
```

Comment:
Might I suggest JSON? Less validation and processing required.

Reply:
Hmm, as JSON is more modern, and is more flexible it would be a nice thing to do.